packages/ua-devtools-evm-hardhat/src/tasks/oapp/wire/index.ts

```diff
@@ -61,11 +61,15 @@ const action: ActionType<TaskArgs> = async (
         )
     )
     // Now sign & send the transactions
-    const signAndSendResult = await hre.run(SUBTASK_LZ_SIGN_AND_SEND, {
+    const signAndSendResult: SignAndSendResult = await hre.run(SUBTASK_LZ_SIGN_AND_SEND, {
         transactions,
         ci,
     } satisfies SignAndSendTaskArgs)
 
+    // Mark the process as unsuccessful if there were any errors (only if it has not yet been marked as such)
+    const [, failed] = signAndSendResult
+    if (failed.length !== 0) process.exitCode = process.exitCode || 1
```

Comment:
Can we please use french braces, even for single line conditional clauses?

```
if (failed.length !== 0) {
    process.exitCode = process.exitCode || 1
}
````

Reply:
Im indiffeferent, but should just be consistent everywhere~

Reply:
Consensus is {}, will do in a seperate PR